### PR TITLE
RHBA-519: Cache AsyncAuditLogProducer in DefaultRegisterableItemsFact…

### PR DESCRIPTION
**Pull request by martinweiler**
…ory to improve performance

### CODE REVIEW

**Comment by mswiderski** [comment]
@martinweiler would it be possible to add simple test - just to make sure we cover it with some verification?

**Comment by martinweiler** [comment]
@mswiderski sure, I'll try to add a test. Do you happen to have a good pointer for an existing test that I could use as a starting point?

**Comment by mswiderski** [comment]
@martinweiler maybe just add simple test case for DefaultRegisterableItemsFactory that will mock things needed to get this to build a new instance and cache it.

**Comment by martinweiler** [comment]
@mswiderski test case added, please review. thanks!

**Comment by mswiderski** [comment]
Jenkins retest this

File: jbpm-runtime-manager/src/test/resources/jbpm.audit.jms.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (mswiderski) [inline jbpm-runtime-manager/src/test/resources/jbpm.audit.jms.properties]: is this intentionally left empty? if so maybe just add a comment about it so it won't be removed by accident

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by martinweiler** [comment]
@mswiderski thanks, let me know in case any other formatting changes are required.

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**